cognite/fdm/config.example.yaml

```diff
@@ -3,7 +3,7 @@ cognite:
   tenant_id:
```

Comment:
Isn't this file outdated with the introductions of the new `settings.toml` + sectets file?

Reply:
yeah definitely. deleted.